internal/services/ingest_test.go

```diff
@@ -214,4 +218,131 @@ func TestIngestPayments(t *testing.T) {
 		assert.NoError(t, err)
 		assert.Equal(t, payments[0].TransactionHash, "abcd")
 	})
+
+	t.Run("test_op_path_payment_send", func(t *testing.T) {
+		err := models.Account.Insert(context.Background(), srcAccount)
+		require.NoError(t, err)
```

Comment:
nitpicking here but I don't really think you require this line. This is not a test to see if models are working. I'd replace it with _ := models.Account.Insert(context.Background(), srcAccount)